docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5637,12 +5659,21 @@ it populates the `MasterArbitrationUpdate` message using the `device_id`,
 also populates the `status` field in the `MasterArbitrationUpdate` according to
 the rules in an [earlier section](#sec-arbitration-updates).
 
-The sender need not specify an `election_id`. If the `election_id` is not
-specified, the sender's `election_id` is considered lower than any
-`election_id`, and the sender will thus never become primary. This way, a
-controller can choose to be a backup controller, in order to avoid
-"flapping" (if a backup controller connects to the switch shortly before the
-actual primary controller, therefore becoming primary temporarily).
+### Unset Election ID
+
+The sender need not specify an `election_id`. If the `election_id` is
+unset, the sender's `election_id` is considered lower than any
+`election_id`, and the sender will thus never become master. This way, a
```

Comment:
Wasn't there a pass through the spec a while back to replace occurrences of `master` with `primary`?  This change seems to undo some of that.

Reply:
thanks for catching this Andy. @smolkaj could you take care of this, I am not sure why you removed instances of `primary` in your PR?

Reply:
@smolkaj friendly ping

Reply:
Ugh... looks like maybe my change was based on an old version of the spec? 

Let me try to revert & reapply the changes.

Reply:
It was easy to fix actually.